modules/xmpp/ChatRoom.js

```diff
@@ -1401,6 +1401,19 @@ export default class ChatRoom extends Listenable {
             errorCallback);
     }
 
+    /**
+     * Adds the key to the presence map, overriding any previous value.
+     * This method is used by jibri.
```

Comment:
Do you want to mark it as deprecated to remind us to remove it once jibri is updated?

Reply:
Done